src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_DelegateCreationExpression.cs

```diff
@@ -21,6 +21,14 @@ public override BoundNode VisitDelegateCreationExpression(BoundDelegateCreationE
                 return new BoundDelegateCreationExpression(node.Syntax, loweredReceiver, methodOpt: null, isExtensionMethod: false, type: node.Type);
             }
 
+            if (node.Argument.Kind == BoundKind.MethodGroup)
+            {
+                var mg = node.Argument as BoundMethodGroup;
+                var method = node.MethodOpt;
+                var receiver = (method.IsStatic && !node.IsExtensionMethod) ? _factory.Type(method.ContainingType) : VisitExpression(mg.ReceiverOpt);
```

Comment:
>_factory.Type(method.ContainingType) [](start = 78, length = 36)

Consider adjusting _factory.Syntax before creating synthetic node so that it would be easier to investigate any issues related to that node and avoid confusing a fellow developer with a "random" syntax associated with the node.